\documentclass[11pt]{article}


%\documentclass[11 pt]{article}
\usepackage{graphicx}
\usepackage{float}

\usepackage[normalem]{ulem}

\usepackage[utf8]{inputenc}

\usepackage[square,sort,comma,numbers]{natbib}
\bibliographystyle{abbrvnat}
\setcitestyle{authoryear,open={(},close={)}} %Citation-related commands

\usepackage{setspace}

\usepackage{pdflscape}
%\doublespacing
\onehalfspacing
\usepackage[dvipsnames]{xcolor}
\usepackage{lineno}
\linenumbers
\usepackage{multicol}
\usepackage{hyperref}
\hypersetup{
	colorlinks=false,
%	linkcolor=blue,
%	filecolor=magenta,      
%	urlcolor=cyan,
	pdftitle={Overleaf Example},
	pdfpagemode=FullScreen,
}

\addtolength{\oddsidemargin}{-2.5cm}
\addtolength{\evensidemargin}{-1.5cm}

\addtolength{\textwidth}{5.5cm}

\addtolength{\topmargin}{-.875in}
\addtolength{\textheight}{1.75in}

% color can be used to apply background shading to table cells only
%\usepackage[table]{xcolor}

% array package and thick rules for tables
\usepackage{array}

% create "+" rule type for thick vertical lines
\newcolumntype{+}{!{\vrule width 2pt}}

% create \thickcline for thick horizontal lines of variable length
\newlength\savedwidth
\newcommand\thickcline[1]{%
  \noalign{\global\savedwidth\arrayrulewidth\global\arrayrulewidth 2pt}%
  \cline{#1}%
  \noalign{\vskip\arrayrulewidth}%
  \noalign{\global\arrayrulewidth\savedwidth}%
}

% \thickhline command for thick horizontal lines that span the table
\newcommand\thickhline{\noalign{\global\savedwidth\arrayrulewidth\global\arrayrulewidth 2pt}%
\hline
\noalign{\global\arrayrulewidth\savedwidth}}



%\author{Pending}


\begin{document}
%\title{Understanding the emergence of viral variants of concern via Bayesian molecular clock model selection}
\begin{flushright}

%\today
\end{flushright}
\begin{center}
	\begin{LARGE}
	\textbf{Supplementary results for: The phylodynamic threshold of measurably evolving populations}
	\end{LARGE}


Ariane Weber$^{1,*}$, Julia Kende$^{2, 3}$, Camila Duitama González$^{3,4}$, Sanni Översti$^{1, \ddagger}$ and Sebastian Duchene$^{3,4, 5,\ddagger, *}$.
\end{center}

$^{1}$ Max Planck Institute of Geoanthropology, Jena, Germany.

$^{2}$ Bioinformatics and Biostatistics Hub, Institut Pasteur, Paris, France.

$^{3}$ Université Paris Cité, Paris, France.

$^{4}$ ED-ID unit, Dept of Computational Biology, Institut Pasteur, Paris, France.

$^{5}$ Peter Doherty Institute for Infection and Immunity, Dept of Microbiology and Immunology, University of Melbourne, Melbourne, Australia.


*email: weber@gea.mpg.de, sduchene@pasteur.fr

$\ddagger$ Equal contribution to the supervision of this work.
\newline
In fig. S\ref{figure:Fig1} we show the posterior estimates of the root height for our simulations of varying sampling window widths. 

\begin{landscape}
	\begin{figure}[H]
		\begin{center}
			\includegraphics[scale=0.7, angle=0]{testing_prior_uncertainty_tree_height_fig_reordered.pdf}
			\caption{Estimates of the age of the root-node for simulations of varying sampling window widths. Each column corresponds to a simulation setting: (a) is for ultrametric trees where all samples are collected at the same point in time (sampling window, sw=0), (b) is for the situation where the sampling window is 10 years (half the expected phylodynamic threshold; sampling window, sw=0.5$\times$ ph.t), (c) is where the sampling window is exactly the expected phylodynamic threshold of 20 years (sw=1$\times$ ph.t). Scenarios (d) and (e) denote sampling windows that are 10 and 100 $\times$ the expected phylodynamic threshold (sw=10$\times$ ph.t and sw=100$\times$ ph.t, respectively). The rows denote example phylogenetic trees and prior configurations where the mean evolutionary rate prior, $M$, is set to the correct value (fist row), an order of magnitude higher (second row), or an order of magnitude lower, last row). Because each simulated tree has a different root height, we subtract the posterior by the truth and divide by the truth. Thus, an value of 0 is the correct value, where as one of 1 means a one-fold overestimation. The dashed line is for values of 0. Each vertical lines are the 95\% posterior credible interval, with 100 replicates for each of the priors on $M$ in table \ref{table:prior_distros} }
			\label{figure:Fig1}
		\end{center}
		%		\centering
	\end{figure}
\end{landscape}


\begin{table}[H]
	\caption{Prior configuration for the mean evolutionary rate, $M$ of the lognormal distribution of branch rates. Note that the mean of the $Gamma$ distribution here is $shape/rate$ and that the true value used to generate the data is $1.5\times10^{-5}$ subs/site/year.} 
	\begin{center} 
		\label{table:prior_distros}
		\begin{tabular}{ c|c|c }
			Mean value of $M$ & Prior configuration & 95\% quantile width / mean $M$\\
			\hline
			$1.5\times10^{-5}$ & $Gamma(shape=1.5, rate=1\times10^5)$ & $3.04$\\
			$1.5\times10^{-5}$ & $Gamma(shape=0.3, rate=2\times10^4)$ & $6.33$\\
			$1.5\times10^{-5}$ & $Gamma(shape=15, rate=1\times10^6)$ & $1.00$\\
			
			$1.5\times10^{-4}$ & $Gamma(shape=1.5, rate=1\times10^4)$ & $3.04$\\
			$1.5\times10^{-4}$ & $Gamma(shape=0.3, rate=2\times10^{3})$ & $6.33$\\
			$1.5\times10^{-4}$ & $Gamma(shape=15, rate=1\times10^{5})$ & $1.00$\\
			
			$1.5\times10^{-6}$ & $Gamma(shape=1.5, rate=1\times10^6)$ & $3.04$\\
			$1.5\times10^{-6}$ & $Gamma(shape=0.3, rate=2\times10^5)$ & $6.33$\\
			$1.5\times10^{-6}$ & $Gamma(shape=15, rate=1\times10^7)$ & $1.00$\\
		\end{tabular}
	\end{center}	%\vspace{-0.4cm}
\end{table}


\begin{figure}[H]
    \begin{center}
        \includegraphics[scale=0.7, angle=0]{empirical_results_tree_height.pdf}
        \caption{Results from empirical analyses of Hepatitis B virus (HBV) ancient DNA data. The phylogenetic trees correspond to highest clade credibility trees from three analyses where the data were subsampled to include an increasing number of ancient samples. First, we consider a data set for which the samples are 95\% modern and the remaining 5\% being the most ancient. Then, we reduce the number of modern samples to 50\%, 25\% and 10\%, and the rest being ancient. Note that the sampling window is constant because we always retain the most ancient samples. In all cases the data sets consist of 100 genome samples. The violin plots show the posterior distribution of the tree height in blue and its corresponding prior in orange.}
        \label{figure:Fig2}
    \end{center}
    %		\centering
\end{figure}






\end{document}